src/metrics.ts

```diff
@@ -107,13 +123,48 @@ export class MetricsProcessor implements MetricsProcessorInterface {
       variation,
       count: 0,
     };
+    this.storeEvaluationAnalytic(event);
+    this.storeTargetAnalytic(target);
+  }
+
+  private storeTargetAnalytic(target: Target) {
```

Comment:
Totally not an issue, but it's a good practice to add return types
```suggestion
  private storeTargetAnalytic(target: Target): void {
```